app/assets/main/react_components/User/KanbanActionContainer.jsx

```diff
@@ -104,7 +104,20 @@ const KanbanActionContainer = ({ collapsed, setCollapsed, categories }) => {
       );
       //Add moved item to second column. Helpfunction in context desides if new categoryBadge should be created or just change status and color on subitem
       const destItems = setCategoryBadgesOnDrag(movedItem, destColumn.items);
-      setCategoryBadges([...destItems]);
+      const sortedDestItems = destItems.map((category) => {
+        return {
+          ...category,
+          userActionsArray: [
+            ...category.userActionsArray.filter(
+              (action) => action.status == true
+            ),
+            ...category.userActionsArray.filter(
+              (action) => action.status == false
+            ),
+          ],
+        };
+      });
+      setCategoryBadges([...sortedDestItems]);
```

Comment:
Nice! Suggestion for less lines of code is to use a lodash function. I used one to order the action of the month first in the carousell. Checkout the branch!